content/clients/_index.md

```diff
@@ -94,47 +93,49 @@ justify = "center"
         <img src="/images/clients/androidtv.svg"/>
         <div class="bannerbox">
             <p class="banner">Android TV</p>
-            <p class="subbanner">Official</p>
         </div>
         <a href="https://play.google.com/store/apps/details?id=org.jellyfin.androidtv" class="button button__accent button__card">Go to Play Store</a>
     </div>
     <div class="card">
         <img src="/images/clients/firetv.svg"/>
         <div class="bannerbox">
             <p class="banner">Fire TV</p>
-            <p class="subbanner">Official</p>
         </div>
         <a href="https://www.amazon.com/gp/aw/d/B07TX7Z725" class="button button__accent button__card">Go to Amazon</a>
     </div>
+    <div class="card">
+        <img src="/images/clients/google-cast.svg"/>
+        <p class="banner">Chromecast and Google Cast devices</p>
```

Comment:
```suggestion
        <p class="banner">Google Cast</p>
```

Reply:
@dkanada I know it makes sense to call it Google Cast alone, in fact that's how it was when I added it first (but decided to change it later). The problem with Google Cast is that it's called everywhere as "Chromecast" (by Netflix, etc) but *it's not only Chromecast, it's a myriad of devices*. This is more a Google problem, because they don't know how to make the brand recognisable :P.

I'm fine with either, just want it to be recognisable for 100% of the people who has one Google Cast device, so maybe the solution is to use subbaners here as well? So ``Chromecast`` as banner and ``Google Cast`` as subbanner, ripping off **devices** completely.

Reply:
Maybe make it a link with hint on hover saying "Includes Chromecast"? Like described here: https://www.w3schools.com/howto/howto_css_tooltip.asp

Reply:
I think we should stick to Google Cast, since (according to Google), it is the name of the ecosystem of devices (one of which is Chromecast).

The icon should be pretty recognizable and we should generally respect the platform holders' guidelines and naming conventions.

Reply:
@JustAMan I like the tooltip idea, @MrTimscampi I think I will apply it, so we have the best of the two worlds.

I know Google calls it this way, but is not listed everywhere that way. As a brand, Chromecast is more recognised.